torch/_functorch/aot_autograd.py

```diff
@@ -2851,9 +2846,54 @@ def functional_call(*args, **kwargs):
         bw_compiler = fw_compiler
 
     full_args = []
+    # First, the params
     full_args.extend(params_flat)
+
+    aot_autograd_arg_pos_to_source = []
+    # Then, the params 1:1 mapped sources, if relevant.
+    if hasattr(mod, "_nn_module_sources"):
```

Comment:
This isn't the source of an nn module? It's the source of a parameter no?

Reply:
Top level params would just have a name like `weight` or `buf` or whatever. It is a small case.

Reply:
It is confusing for both parameters and modules to be in a map called `_nn_module_sources`!